composer.json

```diff
@@ -27,7 +27,6 @@
     "symfony/messenger": "^6.4"
   },
   "require-dev": {
-    "roave/security-advisories": "dev-latest",
```

Comment:
I think this change can be reverted now.

Reply:
Yes, it works now - thanks again for the hint 👍